.contributing/code-of-conduct.md

```diff
@@ -63,9 +63,21 @@ DO attempt to communicate effectively when creating tasks/issues/etc.
   - try and read what you've written with the mindset of someone who doesn't have all the context that you currently do 
 
 #### Standups
-- goal is to get the team up to speed on where you are in your workflow. And identify anything blocking you.
-- with the goal of preventing discussions that could waste someones time
-  - try to push any discussions to post-standup 
-  - all team members please feel free to transform into stand up police should a discussion that is wasting your time erupt
-    - stand up police have all rights to flag conversations as post-standup items
+
+DO share your progress with the team
+- Get the team up to speed on where you are in your workflow.
+- Identify anything blocking you.
+- Finish with a "maptivation" statement: 
+  - draw us a map of where we're going 
```

Comment:
I think it should be "draw us a map of where you're going"

Reply:
Hey mate, thanks for your input!  I intentionally had it a little weird sounding, to emphasise the point that you may have goals you want to reach for the day but they are part of where "we" are going today.  

What's your reasoning behind thinking it should be "draw us a map of where you're going"?

Reply:
Hey thanks for the Q, I could've added a bit more.

They are getting the team up to speed with where _they_ are and what _they_ have planned, so I see the next part of that for them to get us excited on where _they_ are going.

I think it can be read and understood with "we're", but I do find it a bit more clear as "you're" as it points to drawing a map of what they are doing today instead of what the team is doing today.

I support either, you've got my approval 👍

Reply:
Beautiful, thanks mate!  I definitely see where you're coming from.  I'll push it through now and add an RTC for when Tam comes back to have a chat and get consensus.  Even though it's all of like 4 letters difference I think it's a distinction worth discussing.  Appreciate the discussion!